content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -5,7 +5,7 @@ weight: 2
 spelling: cSpell:ignore autoconfiguration Autoconfiguration Dotel HSET javaagent serverlessapis Servlet servlet
 ---
 
-> <h3>NOTE: subject to change!</h3>
+> ### NOTE: subject to change!
```

Comment:
This change should be reverted so that the entry doesn't appear in the right-side TOC

Reply:
Preview: https://deploy-preview-1482--opentelemetry.netlify.app/docs/instrumentation/java/automatic/agent-config/

Reply:
@trask, @mateuszrzeszutek et all: maybe the note itself can be dropped?

Reply:
I'm good with dropping the note. All of the experimental configuration options have the word `experimental` in them now (which wasn't the case when this note was originally written).

Reply:
will update accordingly!

Reply:
I removed the note with fe9f612